docker/docker-compose.suite-ci.yml

```
@@ -7,7 +7,7 @@ services:
       - XDG_RUNTIME_DIR=/var/tmp
     network_mode: bridge # this makes docker reuse existing networks
   test-run:
-    image: cypress/included:6.4.0
+    image: gitlab.com/satoshilabs/dependency_proxy/containers/cypress/included:6.4.0
```

```suggestion
    # This file is only ever used in CI. Use docker proxy directly.
    image: gitlab.com/satoshilabs/dependency_proxy/containers/cypress/included:6.4.0
```

---

What do you mean by `Use docker proxy directly.`?

---

The fact that we're not using the proxy conditionally as we do in all the other docker-compose files. I `unconditionally` in place of `directly` might be clearer. .)

---

I still do not get it. Why do we worry about it it runs on CI only?

---

> Why do we worry about if it runs on CI only?

We don't. At least that's what the comment is trying to convey (and apparently isn't much successful in that 😅) If I got one moer chance to reword it I'd write

```suggestion
    # This file is only ever used in CI. Don't bother with conditional proxying.
    image: gitlab.com/satoshilabs/dependency_proxy/containers/cypress/included:6.4.0
```

Anyway this is a totally minor nitpick. @vdovhanych please feel free to decide this yourself in either direction and resolve this thread.

---

Yeah it seems to me it confuses the user more than adding anything to it :laughing:. But I agree it is minor, let's leave it up to @vdovhanych :)

---

I would say this is really not needed, the file itself is named -ci so I don't see the point in stating that there.